__tests__/app.test.js

```diff
@@ -610,15 +585,15 @@ describe('DELETE /api/comments/:comment_id', () => {
 
     test('number of comments should decrease by one after successful deletion', () => {
         return request(app)
-        .delete('/api/comments/5')  // comment 5 refers to article 1
+        .delete('/api/comments/5')
         .expect(204)
         .then(() => {
             return request(app)
             .get('/api/articles/1/comments')
             .expect(200);
         })
         .then(({ body }) => {
-            expect(body.comments.length).toBe(10);  // starts with 11 comments
```

Comment:
As this is related to the previous ticket, please ensure this is dealt with on that branch as it isn't to do with this branch and this ticket